fineract-provider/src/main/java/org/apache/fineract/portfolio/group/api/GroupsApiResource.java

```diff
@@ -508,4 +524,64 @@ public String postGroupTemplate(@FormDataParam("file") InputStream uploadedInput
                 uploadedInputStream,fileDetail,locale,dateFormat);
         return this.toApiJsonSerializer.serialize(importDocumentId);
     }
+
+    @GET
+    @Path("{groupId}/glimaccounts")
+    @Consumes({ MediaType.APPLICATION_JSON })
+    @Produces({ MediaType.APPLICATION_JSON })
+    public String retrieveglimAccounts(@PathParam("groupId") final Long groupId, @QueryParam("parentLoanAccountNo")final String parentLoanAccountNo,  @Context final UriInfo uriInfo)
+    {
+         this.context.authenticatedUser().validateHasReadPermission("GROUP");
+         List<GLIMContainer> glimContainer;
```

Comment:
Try using  Collections.emptyList() to initialize container variables, it helps to avoid ugly null pointer exceptions